code/datums/traits.dm

```diff
@@ -526,20 +526,22 @@
 
 /datum/trait/mildly_mutated
 	name = "Mildly Mutated"
-	desc = "A random mutation in your gene pool starts activated."
+	desc = "A random mutation in your gene pool starts activated and immune to mutadone."
 	id = "mildly_mutated"
 	icon_state = "mildly_mutatedB"
-	points = 0
+	points = 1
 	category = list("genetics")
 	afterlife_blacklisted = TRUE
 	disability_type = TRAIT_DISABILITY_MINOR
 	disability_name = "Genetic Deviation"
-	disability_desc = "Minor alteration from baseline genetic sequence"
+	disability_desc = "Minor reinforced alteration from baseline genetic sequence"
 
 	onAdd(var/mob/owner)
 		var/datum/bioHolder/B = owner.bioHolder
 		var/datum/bioEffect/E = pick(B.effectPool)
 		B.ActivatePoolEffect(B.effectPool[E], 1, 0)
+		E.curable_by_mutadone = FALSE
+		E.name = "Reinforced " + E.name
 		SPAWN (1 SECOND) // This DOES NOT WORK at round start unless delayed but somehow the trait part is logged??
 			logTheThing(LOG_COMBAT, owner, "gets the bioeffect [E] from the trait [name].")
```

Comment:
```
/datum/trait/mildly_mutated
	name = "Mildly Mutated"
	desc = "A random mutation in your gene pool starts activated."
	id = "mildly_mutated"
	icon_state = "mildly_mutatedB"
	points = 0
	category = list("genetics")
	afterlife_blacklisted = TRUE
	disability_type = TRAIT_DISABILITY_MINOR
	disability_name = "Genetic Deviation"
	disability_desc = "Minor alteration from baseline genetic sequence"
	var/datum/bioEffect/activated_effect = null

	onAdd(var/mob/owner)
		var/datum/bioHolder/B = owner.bioHolder
		var/datum/bioEffect/E = pick(B.effectPool)
		var/msgGain = E.msgGain
		E.msgGain = ""
		var/datum/bioEffect/activated = B.ActivatePoolEffect(B.effectPool[E], 1, 0)
		E.msgGain = msgGain
		if (istype(activated))
			src.activated_effect = activated
			SPAWN (1 SECOND) // This DOES NOT WORK at round start unless delayed but somehow the trait part is logged??
				logTheThing(LOG_COMBAT, owner, "gets the bioeffect [E] from the trait [name].")

	onRemove(mob/owner)
		if (!isnull(src.activated_effect))
			// TODO: check the owner has the bioeffect still
			var/msgLose = src.activated_effect.msgLose
			src.activated_effect.msgLose = ""
			owner.bioHolder.DeactivatePoolEffect(src.activated_effect)
			src.activated_effect.msgLose = msgLose
```
Something like this? Would suppress the message shown to people at roundstart, but I'd argue that's a feature.

Reply:
EDIT: we're going a different way with it, I think.